### ISSUE TITLE
<Content Update: Intro pages> from <Benefits Disability Experience 526ez>

### ISSUE THREAD

**Issue by RakshindaAslam** [department-of-veterans-affairs/va.gov-team#57843]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [x] Making revisions to one or more existing VA.gov pages
- [ ] Something else 
 
 *Give a brief description of what your team needs support with.*
These content changes are focused on changes to the introduction/landing page for 526ez form. These changes are covering:
- Part of updates to make the 526ez digital form current with the new paper version
- Changes to lower the burden for completing these forms by setting expectations and improving  information to help the veteran complete the form.

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [ ] Yes
  - [x] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- [Link to Research Plan](https://github.com/Nyanteea/va.gov-team/blob/master/products/disability/526ez/Research/2023-05-526ezLandingPagesRedesign/research-plan.md)
- [Link to Discussion Guide ](https://github.com/Nyanteea/va.gov-team/blob/master/products/disability/526ez/Research/2023-05-526ezLandingPagesRedesign/conversation-guide.md)
- Link to product outline: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/README.md
- Link to or attach user flows: https://sketch.com/s/0fa92d93-0841-4c62-8112-c9c528204092
- Link to prototype or mockups: https://sketch.com/s/0fa92d93-0841-4c62-8112-c9c528204092
- Link to any epics/issues that may be helpful: https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/55513
https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/56607
- Link to original content in Google doc: https://docs.google.com/document/d/1FeN68e-udsPEd_bTUmMSUiiB5UpJ34tFoXiZ657jvdY/edit?usp=sharing
- Link to SOT on Github: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/design/introduction%20page%20content.md

## Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date: 
- Estimated staging review date:
- Content, Accessibility, and IA work needed by: 05/10/2023

## Do you plan to bring this to an upcoming content office hours session?
- [x] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [ ] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: 
-	OCTO-DE product owner: Matt Self
-	Product manager: Rakshinda Aslam
-	Designer: Lydia Hooper
-	FE engineer: Christine Cereca
-	Product/team Slack channel:https://dsva.slack.com/archives/C04KW0B46N5

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by lydiahooper** [comment]
I am working with Beth on content, but want to make sure y'all are aware that included in this work is also changes to IA per the user flow in the Sketch file linked to. We are removing the pre-authentication /start page and adding two pages right after authentication to replace it. Please let me know if you have any questions!

**Comment by lydiahooper** [comment]
After getting Beth's edits on May 9, and meeting with Shannon and Nyantee on May 10 to clarify research objectives, I've updated Github SOT and design prototypes with suggested changes. Kept some of the language it was suggested we cut so we can conduct more research to better inform our decisions. We'll also discuss will VBA on May 18 so we can make sure we are meeting legal requirements and properly syncing with the paper/PDF version of the form.

**Comment by bethpottsVADEPO** [comment]
@lydiahooper I know you already have it, but I'm uploading here the Word doc with my edits to make sure they're attached to a ticket. I added in placeholder text for the time it takes VA to make a decision on a claim--thanks for calling out that inaccuracy.

[https://app.zenhub.com/files/133843125/1597c1f1-df60-4ccc-900f-b1a2da54b765/download](https://app.zenhub.com/files/133843125/1597c1f1-df60-4ccc-900f-b1a2da54b765/download)

And here's some feedback on the new screens for "Service end date content" (if yes to active duty):

- I'd recommend shortening this: What’s the date or anticipated date your active duty service ends? (This may not be the same date as the date you are released from your contract or the date your obligation term of service ends.)
To this: When do you plan to separate from active duty? 
**Note:** This may be a different date from when you're released from your contract or when your service obligation ends.
- When we talk about selecting a button, our style is to bold the label for the button rather than using quote marks, so just bold **Continue** on any screen where you're directing the user to click that button. No single quote marks. Also, we don't bold entire sentences for emphasis. And I've formed a contraction for "you'll" per our conversational style. I'd recommend this: Select **Continue** to start now and declare your intent to file. This will reserve a potential effective date for when you could start getting benefits. You'll have 1 year from the day you submit your intent to file to complete your application.
- For the "You have x days left content" - that sentence is too long and doesn't align with our plain language guidance about short sentences. I'd recommend: You have x days left (until x) to file your claim under the BDD program.
- Blue box (add "planned" like this and remove the period following the text link): Based on your planned separation date...
- Blue box: After this date, you can still apply for benefits using VA Form 21-526EZ. We encourage you to start filling out this application as soon as you can to declare your intent to file. This reserves a potential effective date for when you could start getting benefits. You have 1 year from the date you submit your intent to file to complete your application.
- Self-Assessment alert: 1/ We don't talk about completing a form in terms of "preparing" it. The first word of the header for that alert should be changed to either "Complete" or "Fill out." I'd also change "your" to "a" in that header. Something for us to consider is whether each time we mention the Self-Assessment we need to use the full name of the form--it's kind of long in the header and perhaps could be generalized (lowercased) and shortened to something like "Complete a separation health self-assessment" 2/ The SOT differs from the sketch language and includes "service treatment records for the current period of service as well as," which shouldn't be included in the alert. Just wanted to call that out to be fixed in SOT. 3/ This line should be changed to active voice: Your claim will be delayed until we receive this form. (We can't make a decision on your claim until we receive this form.) 4/ There's too much information in this alert, especially following the content-heavy blue box that precedes it. I'd recommend cutting the part about "we recommend that you download and fill out this form on a desktop computer." If you choose to keep it, I'd reword to avoid using "form" 4 times in the course of 2 sentences; I'd also stay consistent calling the 526 an application and call the self-assessment a form  or it can get pretty confusing (we'll have to clean this up throughout at some point): "We estimate that it will take you at least 30 minutes to complete this self-assessment. Download and fill out the self-assessment form on a desktop computer or laptop then return to this application to upload it."

**Comment by lydiahooper** [comment]
I've made changes based on the previous comment as well as Beth's suggestion to use plain language guidelines to improve a couple sections we plan to test (that she'd recommended we cut). 

Our VBA SME (Alejandro Mendiolaflores) reviewed revised content yesterday and had some concerns based on policy, the most significant of which is about helping vets discern the difference between filing a CFI vs. supplemental claim. So I changed some of this content based on what's on the [supplemental claims page](https://www.va.gov/decision-reviews/supplemental-claim/) on va.gov.

All of these changes are now reflected in the SOT on Github, and they will be reflected in the prototypes in Sketch later today.

**Comment by jennymayoco** [comment]
Hi @lydiahooper @RakshindaAslam - I wanted to follow up with the IA feedback from Monday's sync with the team. I paired some of the feedback with IA heuristics, initially focusing on the user flows. I realize any additional IA support may come after research. Let me know if you have any questions in the meantime. Thanks!

#### IA Heuristics Table - User Flows
| Standard | This product  |
| --- | --- | 
| A user flow has valid or appropriate closure, and doesn't result in a dead end where a user can't complete a task due to a functional issue (i.e. blank page, missing field or CTA, 404 error). | Clear next steps. If no to active duty, continue to 526 normal flow. If yes to active duty, continue through BDD flow |
| A user flow provides content or navigation to help users understand where to go or what to do next. | N/a |
| A user flow includes a prominently placed CTA that doesn't lead the user out of the primary flow without notifying them or providing a clear way for them to return to the original flow. | Flagging this because the H1 changes (although it’s not CTA). Team might implement a question in research to see if this causes any disruption to the flow for participants |
| A user flow doesn't have repetitive or redundant steps. | N/a |
| The primary CTA is clear and/or prominently displayed within the flow or content page. | N/a |
| Error messages include a path to resolution and follow guidance in the error messages dictionary and error messages style guide. | N/a |
| Your content or feature supports all necessary user flows and scenarios. | Can non-Veterans fill this form out for someone who is a Veteran or active duty member? If they’re logged in as themselves or logged in as the Veteran? |

CC @kristinoletmuskat @mnorthuis for visibility

**Comment by jennymayoco** [comment]
Hi @RakshindaAslam @lydiahooper - I wanted to check in and see if there was any additional IA support needed based on the usability test findings?

CC @kristinoletmuskat

**Comment by RakshindaAslam** [comment]
@jennymayoco - thanks for following up. This ticket is on hold while @lydiahooper is OOO. We will get back to you in 2nd week of Aug after Lydia gets back from leave.

**Comment by jennymayoco** [comment]
@RakshindaAslam No worries! Thank you for the update.

**Comment by jennymayoco** [comment]
Hi @RakshindaAslam @lydiahooper I will be OOO 8/30-9/5 and so I am looping in @kristinoletmuskat in case of any IA updates. Thanks!

**Comment by lydiahooper** [comment]
Closing this ticket per conversation with @shannonkford
